example2.c

```diff
@@ -2,14 +2,88 @@
 
 #include "vesuvius-c.h"
 
-int main(int argc, char** argv) {
-  //Scroll 1 20230205180739
-  volume* vol = vs_vol_new((s32[3]){14376,7888,8096}, false, true, true, "./cache", 20230205180739);
+int test_volume_load() {
+    chunk* mychunk;
+
+    void* metadata_buf;
+    char url[1024] = "https://dl.ash2txt.org/full-scrolls/Scroll1/PHercParis4.volpkg/volumes_zarr_standardized/54keV_7.91um_Scroll1A.zarr/0/.zarray";
+    if (vs_download(url, &metadata_buf) <= 0) {
+        return 1;
+    }
+    zarr_metadata metadata;
+    if (vs_zarr_parse_metadata(metadata_buf, &metadata)) {
+        return 1;
+    }
+    void* compressed_buf;
+    strncpy(url,"https://dl.ash2txt.org/full-scrolls/Scroll1/PHercParis4.volpkg/volumes_zarr_standardized/54keV_7.91um_Scroll1A.zarr/0/50/30/30",1023);
```

Comment:
... ah, okay, this is why not []. I don't love this; I think I would have two or more strings. Also this way they can be at the start of the example together for if people want to edit one or more at the same time - having it in the middle of the function is a bit weird.

Reply:
addressed